Makefile

```diff
@@ -15,7 +11,10 @@ fmt:
 	@git diff --exit-code .
 
 build:
-	go build ${BUILDFLAGS} ${LDFLAGS} -o ./bin/osdctl main.go
+	goreleaser --snapshot --skip-publish --rm-dist
```

Comment:
The `goreleaser --snapshot --skip-publish --rm-dist` is used for a dry-run of the release locally. Is this what you want to do or just build? 

Otherwise, if you want to just build, you can do `goreleaser build --rm-dist`

Reply:
Thanks, I somehow missed that!